.github/workflows/default.yml

```diff
@@ -0,0 +1,32 @@
+name: default
```

Comment:
@lbn  I have doubts about the pipeline, jobs, steps naming. Maybe you can suggest something.